tests/cypress/cypress/e2e/accessmatch/ccep-details.cy.js

```diff
@@ -5,15 +5,24 @@
     Header text,
     and a segment of body paragraph 
     
+
 */
 
 describe("Unauthenticated user tests the CCEP Details Page", () => {
   it("Should test CCEP Details page for unauthenticated user", () => {
     cy.visit("/ccep-details");
-    cy.contains("CCEP Details");
-    cy.contains("Important Fine Print:");
-    cy.contains(
+
+    //Verifying page title and paragraph
+    cy.get(".page-title > .field").contains("CCEP Details");
+    cy.get(".clearfix > h2").contains("Important Fine Print:");
+    cy.get(".clearfix > ul > :nth-child(1)").contains(
       "All CSSN members (students, faculty, staff, or CI professionals) seeking CCEP rewards need to be affiliated"
     );
+
+    //Verifying Join CCEP Button  Test fails disabling for now
```

Comment:
Not sure why but even though there is a button it fails when the below lines are ran

Reply:
Was able to fix!